remote/remote_test.go

```diff
@@ -1539,6 +1560,48 @@ func testImage(t *testing.T, when spec.G, it spec.S) {
 				h.AssertEq(t, image.CheckReadAccess(), false)
 			})
 		})
+
+		when("using custom handler", func() {
+			it.Before(func() {
+				os.Setenv("DOCKER_CONFIG", customRegistry.DockerDirectory)
+			})
+
+			it.After(func() {
+				os.Setenv("DOCKER_CONFIG", dockerRegistry.DockerDirectory)
+			})
+
+			when("image has read/write access", func() {
```

Comment:
```suggestion
			when("image has read and write access", func() {
```

Reply:
Why do we only call `CheckReadAccess` instead of `CheckReadWriteAccess`?
Moreover, maybe it's better that `CheckReadWriteAccess` will return 2 boolean values for the read and write access separately.

Reply:
Added in the following [commit](https://github.com/buildpacks/imgutil/pull/125/commits/87d5ba9fc355d30923e2cf672f08e02c9de908fd)